gp-inventory/gpi-inventory-shortcode.php

```diff
@@ -16,13 +16,14 @@
  * @todo
  * - Add support for excluding field parameter and showing a consolidated list of all inventories.
  */
-add_filter( 'gform_shortcode_inventory', function ( $output, $atts, $content ) {
+add_filter( 'gform_shortcode_inventory', 'gpi_inventory_shortcode', 10, 3 );
+function gpi_inventory_shortcode( $output, $atts, $content ) {
```

Comment:
Is this change mainly for the sake of unhooking it?

Reply:
It's actually used on line 68 to generate the markup for each individual scope value: https://github.com/gravitywiz/snippet-library/pull/261/files#diff-0cf31022316090e53ef698274fff052e5e424f3c418ffa7334c8b6bd08c47809R68